proxies/dokarkiv-proxy/src/main/java/no/nav/testnav/proxies/dokarkivproxy/StatusController.java

```diff
@@ -0,0 +1,63 @@
+package no.nav.testnav.proxies.dokarkivproxy;
+
+import org.springframework.http.MediaType;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.reactive.function.client.WebClient;
+import reactor.core.publisher.Mono;
+
+import java.util.Map;
+import java.util.concurrent.ConcurrentHashMap;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+@RestController
+public class StatusController {
+    final String TEAM_DOKARKIV = "Team Dokumentløsninger";
+
+    @GetMapping(value = "/internal/status", produces = MediaType.APPLICATION_JSON_VALUE)
+    public Map<String, Map<String, String>> getStatus() {
+        var statusWebClient = WebClient.builder().build();
+
+        return Stream.of("q1", "q2", "q4", "q5", "qx", "t0", "t1", "t2", "t3", "t4", "t5", "t13")
```

Comment:
Hvis det ikke blir for mye jobb, kunne du hentet disse fra miljo-service, slik at den alltid er oppdatert med de gyldige dolly miljøene? 🤓